src/utils/vr-caps-detect.js

```diff
@@ -43,6 +43,8 @@ export async function getAvailableVREntryTypes() {
   const isSamsungBrowser = browser.name === "chrome" && navigator.userAgent.match(/SamsungBrowser/);
   const isOculusBrowser = navigator.userAgent.match(/Oculus/);
   const isDaydreamCapableBrowser = !!(isWebVRCapableBrowser && browser.name === "chrome" && !isSamsungBrowser);
+  const isIPhone = navigator.userAgent.match(/iPhone/);
```

Comment:
Would we want to guard against other iDevices? See this regex @robertlong constructed previously: https://github.com/mozilla/hubs/blob/5a8c9635aa1414e5dcabf65d9709b880ce1a6cba/src/utils/ios-audio-context-fix.js#L5

Reply:
:+1:  I don't own an iPad, but I'd assume the bug exists there as well?

Reply:
we already have `https://www.npmjs.com/package/mobile-detect` in our packages, perhaps that's a good way to check this? or `https://www.npmjs.com/package/device-detect`